src/probe_modules/module_upnp.c

```diff
@@ -176,14 +185,8 @@ void upnp_process_packet(const u_char *packet,
 			      (void *)&udp[1], 0);
 
 		free(s);
-#if 0
 	} else if (ip_hdr->ip_p == IPPROTO_ICMP) {
-		struct icmp *icmp = (struct icmp *) ((char *) ip_hdr + ip_hdr->ip_hl * 4);
-		struct ip *ip_inner = (struct ip *) ((char *) icmp + ICMP_UNREACH_HEADER_SIZE);
-		// ICMP unreach comes from another server (not the one we sent a probe to);
-		// But we will fix up saddr to be who we sent the probe to, in case you care.
-		fs_modify_string(fs, "saddr", make_ip_str(ip_inner->ip_dst.s_addr), 1);
-		fs_add_string(fs, "classification", (char*) "icmp-unreach", 0);
+		fs_add_constchar(fs, "classification", "icmp-unreach");
```

Comment:
Are we sure it's an unreach at this point and not another ICMP type? Why isn't this added by `fs_populate_icmp_from_iphdr)`?